src/lib/accessibility/Behaviors/Menu/menuBehavior.ts

```diff
@@ -21,6 +22,8 @@ const menuBehavior: Accessibility = (props: any) => ({
       isCircularNavigation: true,
       preventDefaultWhenHandled: true,
       shouldFocusFirstElementWhenReceivedFocus: true,
+      // TODO: check if this should be applied to other behaviors as well
```

Comment:
probably, we don't need this comment here :)

Reply:
I left it intentionally for your guys, as I don't think we had final decision whether we wanted to have this in the other behaviors.

Reply:
@sophieH29 @jurokapsiar should we add the direction prop to the toolbarBehavior, tabListBehavior as well, cus we have circular navigation there as well?

Reply:
yes, we probably will add it in the future when needed. But don't see any reason to have this comment here.